src/components/ContentList.tsx

```diff
@@ -1,23 +1,36 @@
-import { SimpleGrid, Box, Text } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 import { ROUTES } from '../constants/routes';
+import { Content } from './Main/MainPageContent';
+import { SimpleGrid, Box, Text } from '@chakra-ui/react';
 
-type contents = { title: string; _id: string }[];
+type ContentListProps = {
+  contents: Content[];
+  spacing?: number;
+  borderRadius?: string;
+  width?: string;
+  gridColumns?: { base?: number; md?: number };
+};
 
-const ContentList = ({ contents }: { contents: contents }) => {
+const ContentList = ({
+  contents,
+  spacing = 2,
+  borderRadius = '10',
+  width = '100%',
+  gridColumns = { base: 2, md: 3 },
+}: ContentListProps) => {
   return (
-    <SimpleGrid w="100%" columns={2} spacing={2}>
-      {contents.map(({ title, _id }) => (
+    <SimpleGrid w={width} columns={gridColumns} spacing={spacing}>
+      {contents.map(({ title, _id, image }) => (
         <Box
-          bgImg={'https://bit.ly/dan-abramov'}
+          bgImg={image ? image : 'https://via.placeholder.com/150'}
```

Comment:
```suggestion
          bgImg={image || 'https://via.placeholder.com/150'}
```
이런 식으로 수정하면 좋을 것 같아요!!